autogen/main/versions.tf.tmpl

```diff
@@ -38,7 +38,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.47.0, < 5.0"
+      version = ">= 4.49.0, < 5.0"
```

Comment:
I'm not sure if we consider a required provider version bump a breaking change, since it's a minor bump I think probably not?

Reply:
I'm also a bit confused by the CI failures - it seems like the provider version bump is being ignored, but I don't see any lock files in the repo to update (seems like just updating the `versions.tf.tmpl` worked fine for [#15](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1513) :thinking:)